Tests/Marketplace/marketplace_services.py

```diff
@@ -555,6 +555,50 @@ def _get_certification(support_type, certification=None):
         else:
             return ""
 
+    @staticmethod
+    def _get_search_rank(tags, certification, content_items):
+        """ Returns pack search rank.
+
+        The initial value is 0
+        In case the pack has the tag Featured, its search rank will increase by 10
+        In case the pack was released in the last 30 days, its search rank will increase by 10
+        In case the pack is certified, its search rank will increase by 10
+        In case all the pack's integration are deprecated and there is at least 1 integration in the pack,
+        the pack's search rank will decrease by 10
```

Comment:
```suggestion
        the pack's search rank will decrease by 50
```

Reply:
* Sure
* Yes, currently that's the logic, might change in the not so long future, furthermore, it depends on the `New` tag that is checked and added in each upload so it make sense for now.
* All, you can see that the field was added to all the packs metadata files.